### ISSUE TITLE
include tool to decrypt

### ISSUE THREAD

**Issue by LiEnby** [MC874/MCuri#1]
;) its not that hard to make a marketplace decryptor

**Comment by MC874** [comment]
Hi @LiEnby , thank you for visiting. Is there a way to decrypt worlds without "owning" them? Kinda like skinpacks do.
Regards,
-MC

**Comment by LiEnby** [comment]
Yes ulthough its a bit of an exploit . and mojang could fix it at any time

**Comment by MC874** [comment]
Hi @LiEnby, I hope you don't mind having a conversation with me at Telegram. I want to push this "mcrax" project further. I don't know if I can do-it myself as I don't have much knowledge about it. Your "pillagerbay" project inspired me in many ways, for now, I envision this "mcrax" towards archival purposes. I'm really passionate about it, although, things quite slow for me as I kind of backed out from the community, as it's kind of toxic. Currently, I'm just doing the best I can.

You can contact me at: @emceh. Thank you for your time, I really appreciate it.
-MC
